todo_app/app.py

```diff
@@ -1,6 +1,6 @@
 from flask import Flask, redirect, render_template, request
 from flask.helpers import flash
-import todo_app.data.trello_items as trello
+from todo_app.data.item_service import ItemService
```

Comment:
I like making the name more generic 👍 The really clean switchover in this file is a good sign your interface is well designed!